ci-operator/step-registry/aro-hcp/test/local-upgrade/aro-hcp-test-local-upgrade-commands.sh

```
@@ -58,91 +60,30 @@ if ! yq -e '.defaults.hypershift.sharedIngressImage.registry' config/config.yaml
+  .clouds.dev.environments.${DEPLOY_ENV}.defaults.hypershift.image.repository = strenv(_YQ_HO_REPOSITORY) |
+  .clouds.dev.environments.${DEPLOY_ENV}.defaults.hypershift.image.digest = strenv(_YQ_HO_DIGEST) |
+  .clouds.dev.environments.${DEPLOY_ENV}.defaults.hypershift.sharedIngressImage.registry = strenv(_YQ_SHARED_INGRESS_REGISTRY) |
+  .clouds.dev.environments.${DEPLOY_ENV}.defaults.hypershift.sharedIngressImage.repository = strenv(_YQ_SHARED_INGRESS_REPOSITORY) |
+  .clouds.dev.environments.${DEPLOY_ENV}.defaults.hypershift.sharedIngressImage.digest = strenv(_YQ_SHARED_INGRESS_DIGEST)
+" > "${SHARED_DIR}/hypershift-image-overrides.yaml"
+
+unset _YQ_HO_REGISTRY _YQ_HO_REPOSITORY _YQ_HO_DIGEST
+unset _YQ_SHARED_INGRESS_REGISTRY _YQ_SHARED_INGRESS_REPOSITORY _YQ_SHARED_INGRESS_DIGEST
+
+# shellcheck source=hack/ci/build-config-override.sh
+source hack/ci/build-config-override.sh
```

Shellcheck (0.11.0)

[info] Not following: hack/ci/build-config-override.sh was not specified as input (see shellcheck -x).

(SC1091)